packages/core/database/lib/entity-manager/regular-relations.js

```diff
@@ -248,7 +248,8 @@ const cleanOrderColumns = async ({ id, attribute, db, inverseRelIds, transaction
         const joinTableName = addSchema(joinTable.name);
 
         // raw query as knex doesn't allow updating from a subquery
-        await db.connection
+        await db
+          .getConnection()
```

Comment:
The fix didn't work for me unfortunately because I got a `raw doesn't exist on getConnection()` which it looks to be the source of this issue to begin with:

https://github.com/strapi/strapi/pull/14839/files